apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/PartitionedReadTests.cs

```diff
@@ -29,9 +29,10 @@ public class PartitionedReadTests
         public PartitionedReadTests(PartitionedReadTableFixture fixture) =>
             _fixture = fixture;
 
-        [Fact]
+        [SkippableFact]
         public async Task DistributedReadAsync()
         {
+            Skip.If(_fixture.RunningOnEmulator, "Emulator doesn't support distributed reads yet.");
```

Comment:
Just as an FYI, this test does pass if you use the daily build of the emulator but I gather that this is not available through gcloud yet.